reference/commands/remove.rst

```diff
@@ -94,7 +94,15 @@ pattern including the ``:`` separator of the ``package_id``:
     # in normal situations
 
 
-All the above commands, by default operate in the Conan cache.
+Note that you can filter which packages will be removed using the ``--package-query`` argument:
```

Comment:
can we fully document package query syntax somewhere and provide a link here?
like operators (`AND`, `NOT`, `OR`, `XOR`, `=`, `!=`, etc.), patterns (can I use `build_type=Rel*`?)
also, it's a bit unclear, if it can filter only by settings, but not by options?

Reply:
Good idea! Thanks for the suggestion :)

Reply:
I've clarified that both settings and options work, and will work on adding a syntax reference in another PR